src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -21,50 +22,81 @@ class ProjectCreationModel {
     )
     var sourceLanguageProperty: Language by property()
     var targetLanguageProperty: Language by property()
-    var selectedResource: Collection by property()
-    var selectedAnthology: Collection by property()
-    var selectedBook: Collection by property()
-    var anthologyList: ObservableList<Collection> by property(FXCollections.observableArrayList())
-    var bookList: ObservableList<Collection> by property(FXCollections.observableArrayList())
-
+    var collectionList: ObservableList<Collection> = FXCollections.observableArrayList()
     val languages: ObservableList<Language> = FXCollections.observableArrayList()
-    val resources: ObservableList<Collection> = FXCollections.observableArrayList()
+    var collectionStore: ArrayList<List<Collection>> = ArrayList()
+    var creationDepth: Int = 1
+    var bookDepthReached = false
 
     init {
         creationUseCase.getAllLanguages()
```

Comment:
Not to be overly picky on formatting, but I believe there should be a newline before the first call in a chain
(https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping)